src/annotator/sidebar.js

```diff
@@ -67,6 +69,22 @@ export default class Sidebar {
   constructor(element, eventBus, guest, config = {}) {
     this._emitter = eventBus.createEmitter();
 
+    /**
+     * Tracks which `Guest` has a text selection. It uses the frame identifier
+     * which uniquely identifies each `Guest`. If there is no text selected, the
+     * value is set to `null`.
+     *
+     * @type {null|string}
```

Comment:
I tend to put the null/undefined type last when writing out types as it is the least-distinctive part of the type name.

Reply:
👍🏼 (I have changed this type in the followup PR).